.env.example

```diff
@@ -54,4 +54,7 @@ OTEL_EXPORTER_OTLP_METRICS_ENDPOINT=
 OTEL_EXPORTER_OTLP_METRICS_HEADERS=
 OTEL_EXPORTER_OTLP_METRICS_PROTOCOL=grpc
 OTEL_EXPORTER_DEBUG=true
-OTEL_PREFIX=boxyhq.saas
\ No newline at end of file
+OTEL_PREFIX=boxyhq.saas
+
+
```

Comment:
```suggestion

```

Reply:
Just a small query..
is NEXT_PUBLIC_TERMS_AND_CONDITIONS_URL not defined or just an empty string?

Reply:
Both cases. You can do `if(process.env.NEXT_PUBLIC_TERMS_AND_CONDITIONS_URL)`.